website/alicloud.erb

```diff
@@ -96,6 +96,8 @@
                         </li>
                         <li<%= sidebar_current("docs-alicloud-datasource-pvtz-zone-records") %>>
                             <a href="/docs/providers/alicloud/d/pvtz_zone_records.html">alicloud_pvtz_zone_records</a>
+                        <li<%= sidebar_current("docs-alicloud-datasource-kvstore") %>>
```

Comment:
Please modify 'docs-alicloud-datasource-kvstore' to 'docs-alicloud-datasource-kvstore-instances'